packages/components/src/components/dropdown/dropdown.jsx

```diff
@@ -423,15 +423,22 @@ const Dropdown = ({
                             onKeyPressed={onKeyPressed}
                             value={value}
                             parent_ref={dropdown_ref}
+                            suffix_icon={suffix_icon}
                         />
-                    )}
+                    </div>
+                    {!error && hint && <p className='dc-dropdown__hint'>{hint}</p>}
                 </div>
-                {!error && hint && (
-                    <Text as='p' color='less-prominent' size='xxs' className='dc-dropdown__hint'>
-                        {hint}
-                    </Text>
-                )}
-            </div>
+            </DesktopWrapper>
+            <MobileWrapper>
+                <SelectNative
```

Comment:
@kevinw-binary , why are rendering `SelectNative` within `Dropdown`? We normally handle the conditional `MobileWrapper` and `DesktopWrapper` from outside of the components.